src/js/backend.ts

```diff
@@ -2,7 +2,7 @@ import { BrowserOptions, Transports } from "@sentry/browser";
 import { BrowserBackend } from "@sentry/browser/dist/backend";
 import { BaseBackend, NoopTransport } from "@sentry/core";
 import { Event, EventHint, Severity, Transport } from "@sentry/types";
-import { Alert, YellowBox } from "react-native";
+import * as ReactNative from "react-native";
```

Comment:
L: would not it be better to just add `LogBox`?

Reply:
Fixed, at first I thought it would throw an error if it didn't exist as an export